python/labbox_ephys/api/_workersession.py

```diff
@@ -86,8 +87,19 @@ def handle_message(self, msg):
             self.add_subfeed_watch(
                 watch_name=msg['watchName'],
                 feed_uri=feed_uri,
-                subfeed_name=msg['subfeedName']
+                subfeed_name=msg['subfeedName'],
+                position=msg.get('position', 0)
             )
+        elif type0 == 'addWorkspaceSubfeedWatch':
+            self.add_workspace_subfeed_watch(
+                watch_name=msg['watchName'],
+                position=msg.get('position', 0)
+            )
+        elif type0 == 'appendWorkspaceSubfeedMessage':
```

Comment:
Note: appendDocumentAction is no longer used and will be deleted in a future branch.